docs/versions/index.md

```diff
@@ -0,0 +1,30 @@
+# Versions
+
+!!! info inline end "Version 21.0 is current"
+
+    Read the [release notes for the current version](21.0).
+
+
+The primary mission of HGVS Nomenclature is to facilitate reliable communication of sequence variants, which requires that the HGVS Nomenclature is stable. Nonetheless, modifications will be required from time to time in order to address new scientific needs, resolve inconsistencies, or clarify conventions.  A key goal of the [HVNC](/hvnc/) is to manage such changes in a way that balances the needs of progress and reliable data sharing, and to communicate the changes to the community clearly.
+
+The HGVS Nomenclature uses concepts from [Semantic Versioning](https://semver.org/) to version releases. Each release version will consist of 3 numbers separated by dots in the format x.y.z with the following meanings:
+
+- The "patch" version (z) is reserved for changes that fix or clarify intentions. This is akin to a bug fix.
+- The "minor" version (y) will be used for new features that are consistent with prior versions but should not break them.  For example, adoption of a community proposal that added a new feature would trigger an increase in the minor version.
```

Comment:
I use "syntax" and "expression" to mean different things.

In my intended use, syntax is the rule (e.g., EBNF-like) for a specific expression, like a DNA SNV, and an HGVS expression is a specific variant or instance that is consistent with some syntax (=syntactic rule). I've tried to use them consistently in this way, but can't be sure that I always did so.

(By EBNF-like, I merely mean that we can't put it into an EBNF-based parser because we haven't defined all of the productions.)

Reply:
I agree that syntax is a better term; I always use "syntax" to explain the rule, e.g., the conversion syntax being removed in favor of the delins syntax. For usages of this syntax, I generally use the term "variant description". Although I am a programmer, thus more familiar with the word syntax, it's currently used throughout the website. If "syntax" is too technical of a term, all pages using it should be reworded.

Reply:
So, @ahwagner and @ifokkema, can we agree to use "syntax" to refer to the rule? 

And, "variant description" seems more transparent and accessible to readers than "HGVS expression", so I'm fine with than. 

In other words, let's do what @ifokkema said. Good?

Reply:
Yes, I agree. Updating `HGVS expression` -> `(HGVS)? variant description` throughout the site appears to remain open. Making a commit to address.